docs/data-apps/attribution-modeling/index.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Attribution Modeling"
+sidebar_position: 3
+sidebar_label: "Attribution Modeling"
+---
+
+:::caution
+
+This data app is currently in Private Preview and features may change without notice. 
+
+:::
+
+In today's increasingly complex digital world, users often take multi-channel journeys before converting. Assigning credit across multiple touchpoints is vital to getting an accurate picture of the efficacy of your marketing channels, yet requires merging disparate datasets and running complex calculations.
+
+Our **Attribution modeling** app lowers the barrier to entry for your marketing team through the following features:
+
+- Incremental SQL model in your warehouse for cost-effective computation
+- Choice of first-touch, last-touch, linear and positional methods, with additional filters and transforms available
+- Reports for conversions, revenue, spend and ROAS per channel and campaign
+- Specify your own advertising spend table to generate a Return on Ad Spend analysis
+- Intermediate tables that you can build your own attribution models on top of
+
+
+## Requirements
+
+- [YAUAA enrichment](/docs/enriching-your-data/available-enrichments/yauaa-enrichment/index.md) enabled
+- [Campaign Attribution enrichment](/docs/enriching-your-data/available-enrichments/campaign-attribution-enrichment/index.md) enabled
+- [IP Lookup enrichment](/docs/enriching-your-data/available-enrichments/ip-lookup-enrichment/index.md) enabled
+- [Referrer Parser enrichment](/docs/enriching-your-data/available-enrichments/referrer-parser-enrichment/index.md) enabled
+
+
+- Running the [Snowplow Unified Digital dbt Package](/docs/modeling-your-data/modeling-your-data-with-dbt/dbt-models/dbt-unified-data-model/index.md) with `conversion event(s)` defined, with `snowplow__total_all_conversions` set to `true`
+- Running the [Snowplow Attribution dbt Package](/docs/modeling-your-data/modeling-your-data-with-dbt/dbt-models/dbt-attribution-data-model/index.md) with `conversion event(s)` defined, with `snowplow__total_all_conversions` set to `true`
+- Access to the derived tables granted to the role used when setting up the data app
```

Comment:
This isn't the setup needed on the package?

Reply:
They need to access the intermediate incremental tables to be able to query them and create the csvs

Reply:
Sorry I meant the attribution set up, not table access

Reply:
I still don't get this comment, this is the list of prerequisite for the app itself the model setup is one part of it and i don't mean to expand on it in this list (?)

Reply:
I don't entirely remember... I think I meant that those settings/vars are in the unified package isn't it?

Reply:
Which vars? :D

Reply:
snowplow__total_all_conversions

Reply:
Ah got you. I reworded that section to avoid having to mention vars, it's best if they follow the official guides for those bits in the first place